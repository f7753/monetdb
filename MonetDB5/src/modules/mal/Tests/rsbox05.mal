# Simple test that tests whether `unknown atoms' are currectly printed
# as strings

include rsbox;

b := new(:void, :date);
insert(b, nil, "1999-05-02":date);

rs := rsbox.create(1, 't');
populateColumn(rs, 1, b);

writeNextPart(rs);
