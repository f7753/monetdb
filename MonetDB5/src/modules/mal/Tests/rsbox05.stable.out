stdout of test 'rsbox05` in directory 'src/modules/mal` itself:


# 09:27:17 >  
# 09:27:17 >  Mtimeout -timeout 60 Mserver "--config=/ufs/fabian/scratch/monet5/program/etc/MonetDB5.conf" --debug=10 --set "monet_mod_path=/ufs/fabian/scratch/monet5/program/lib/MonetDB5" --set "gdk_dbfarm=/ufs/fabian/scratch/monet5/program/var/MonetDB5/dbfarm" --set "sql_logdir=/ufs/fabian/scratch/monet5/program/var/MonetDB5/log" --set mapi_port=41325 --set sql_port=52541 --set monet_prompt= --trace --dbname=mTests_src_modules_mal  rsbox05.mal </dev/null
# 09:27:17 >  



# Monet Database Server V4.99.19
# Copyright (c) 1993-2005, CWI. All rights reserved.
# Compiled for i686-redhat-linux-gnu/32bit; dynamically linked.
# config:/ufs/fabian/scratch/monet5/program/etc/MonetDB5.conf
# dbfarm:/ufs/fabian/scratch/monet5/program/var/MonetDB5/dbfarm
# dbname:mTests_src_modules_mal
# Visit http://monetdb.cwi.nl/ for further information.
## Simple test that tests whether `unknown atoms' are currectly printed
## as strings
#
#include rsbox;
#
#b := new(:void, :date);
#insert(b, nil, "1999-05-02":date);
#
#rs := rsbox.create(1, 't');
#populateColumn(rs, 1, b);
#
#writeNextPart(rs);
#
!ERROR: b := new(:void, :date);
!            ^module name expected
!ERROR: insert(b, nil, "1999-05-02":date);
!       ^module name expected
!ERROR: populateColumn(rs, 1, b);
!       ^module name expected
!ERROR: writeNextPart(rs);
!       ^module name expected
function user.main():int;	# 0  unknown (main:int)
    b;	# 1  unknown (b:any)
    ;	# 2  unknown ($2:any)
    rs := rsbox.create(1,'t');	# 3  unknown (rs:any)<-($4:int)($5:chr)
    ;	# 4  unknown ($6:any)
    ;	# 5  unknown ($7:any)
end main;	# 6  unknown 

# 09:27:17 >  
# 09:27:17 >  Done.
# 09:27:17 >  

